dwave_networkx/drawing/zephyr_layout.py

```diff
@@ -52,6 +52,11 @@ def zephyr_layout(G, scale=1., center=None, dim=2):
         Number of dimensions. When dim > 2, all extra dimensions are
         set to 0.
 
+    normalize_kwargs : None or dict (default None)
+        A dict of keyword arguments to be used in a plotting function.  If not
+        None, we will populate the "ax" keyword with matplotlib axes, and the
+        "node_size" and "width" keywords with defaults if they are not set.
```

Comment:
A link to ``draw_lineplot`` for the params would be helpful. Same for the others in Chimera & Pegasus.